server/sonar-server/src/main/java/org/sonar/server/component/ws/ResourcesWs.java

```diff
@@ -29,12 +28,10 @@
   @Override
   public void define(Context context) {
     NewController controller = context.createController("api/resources")
-      .setDescription("Get details about components. Deprecated since 5.4.")
+      .setDescription("Get details about components. Deprecated since 5.4.<br/>" +
```

Comment:
I think the UI is automatically handling this when all actions are deprecated.
Otherwise it could be indeed a good improvement, I thought about it but it's out of scope and I wanted to focus on removing ruby !